### DISCUSSION TITLE
How to define design supports in RFEM6 with the API?

### DISCUSSION THREAD

**Discussion by lucas-ita** [Dlubal-Software/RFEM_Python_Client#354] [Q&A] [answered]
I checked the Dlubal Dev Docs but couldn't find a method that would allow me to configure the member's **design support**. Does such a method exist? Is there any alternative to achieve the same result as the design supports would provide?

**Comment by heetrojivadiya** [comment]
Hey @lucasngbr-eng, thank you for reaching out to us. I have checked that this method design support for members is available in WS. We have planned to implement in Python HLF. We will try to implement it as soon as possible and will let you know.

**Comment by heetrojivadiya** [comment, marked as answer]
Hey @lucasngbr-eng, now you may assign design support to members through API with recent PR #359. You can find the required parameters for it in the [docstrings](https://github.com/Dlubal-Software/RFEM_Python_Client/blob/main/RFEM/TypesForMembers/designSupport.py) and for example you may refer [unittest](https://github.com/Dlubal-Software/RFEM_Python_Client/blob/main/UnitTests/test_designSupport.py) of it.
